### PR TITLE
Added minimak keymaps

### PR DESCRIPTION
**Pull request by SpiritAxolotl**
### Description

> Minimak is designed to give you the benefits of other keyboard layouts without sacrificing your ability to type QWERTY as well.
http://www.minimak.org/
I just added three minimak layouts, all of which can be found in the website I linked above. 
Closes #

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4458865010

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4458884747

File: frontend/static/layouts/_list.json [hunk 1]
```diff
@@ -743,7 +743,7 @@
       "row1": ["ॊऒ", "1ऍ", "2ॅ", "3", "4", "5", "6", "7", "8", "9(", "0)", "-ः", "ऋृ"],
       "row2": ["ौऔ", "ैऐ", "ाआ", "ीई", "ूऊ", "बभ", "हङ", "गघ", "दध", "जझ", "डढ", "़ञ", "ॉऑ"],
       "row3": ["ोओ", "ेए", "्अ", "िइ", "ुउ", "पफ", "रऱ", "कख", "तथ", "चछ", "टठ"],
-      "row4": ["ॆऎ", "ंँ", "मण", "न", "व", "लळ", "सश", ",ष", ".।", "यय़"],
+      "row4": ["ॆऎ", "ंँ", "मण", "न", "व", "लळ", "सश", ",ष", ".।", "यय़"],
```
Comment (Ferotiq) [inline frontend/static/layouts/_list.json:746]: why did this change happen?

Comment (SpiritAxolotl) [reply frontend/static/layouts/_list.json:746]: I have no clue?

Comment (Ferotiq) [reply frontend/static/layouts/_list.json:746]: revert it

**Review by Miodec — APPROVED** [review]

**Pull request merged**